internal/pkg/services/postgresflex/utils/utils_test.go

```diff
@@ -0,0 +1,83 @@
+package utils
```

Comment:
We should also test the other three functions in `utils`: `ValidateStorage`, `ValidateFlavorId` and `LoadFlavorId`

Reply:
Added, for mongodb as well